### PR TITLE
Add a function to display veggie or vegan meals in another style then meaty meals

### PR DESCRIPTION
**Pull request by MaxFleur**
Veggie and vegan meals will be presented using bold markdown. All other meals will be displayed using an italic markdown.

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -246,6 +255,19 @@ def createWisdomString():
...
+    if len(params) < 1:
+        context.bot.send_message(chat_id=update.message.chat_id, text="You know, I can't choose if there is nothing to choose from. Wise words!")
+        return
+    elif len(params) == 1:
+        context.bot.send_message(chat_id=update.message.chat_id, text="How the hell am I supposed to choose when only value is entered? Gosh.")
+        return
+    else:
+        context.bot.send_message(chat_id=update.message.chat_id, text=random.choice(params) + " shall be my answer!")
+    
+
```
Comment (fried-ice) [inline src/main.py:270]: This code is not related.

File: src/main.py [hunk 2]
```diff
@@ -320,6 +342,9 @@ def main():
     wisdomHandler = CommandHandler('wisdom', wisdom)
     updater.dispatcher.add_handler(wisdomHandler)
 
+    chooseHandler = CommandHandler('choose', choose)
+    updater.dispatcher.add_handler(chooseHandler)
+
```
Comment (fried-ice) [inline src/main.py:347]: This code is not related.

File: src/main.py [hunk 3]
```diff
@@ -63,9 +63,18 @@ def mensa(update, context):
...
         context.bot.send_message(chat_id=update.message.chat_id, text="I failed miserably. Disgrace!")
         return
+
     jsonData = json.loads(resp.content)
     for elem in jsonData:
-        context.bot.send_message(chat_id=update.message.chat_id, text=elem["name"])
+        list=elem["notes"]
+        isVeg = 0
+        for veg in list:
+            if "veg" in veg:
```
Comment (fried-ice) [inline src/main.py:72]: We do not know, if there are any notes beside "vegetarisch" and "vegan". False Positives may arise in future. Please filter for the specific keywords only.

File: src/main.py [hunk 4]
```diff
@@ -63,9 +63,18 @@ def mensa(update, context):
     if not resp.ok:
         context.bot.send_message(chat_id=update.message.chat_id, text="I failed miserably. Disgrace!")
         return
+
     jsonData = json.loads(resp.content)
     for elem in jsonData:
-        context.bot.send_message(chat_id=update.message.chat_id, text=elem["name"])
+        list=elem["notes"]
```
Comment (fried-ice) [inline src/main.py:69]: Missing spaces around `=` operator.
Please use expressive variable names. `list` is generic.

File: src/main.py [hunk 5]
```diff
@@ -63,9 +63,18 @@ def mensa(update, context):
...
+        isVeg = 0
+        for veg in list:
+            if "veg" in veg:
+                isVeg = 1
```
Comment (fried-ice) [inline src/main.py:73]: Please use `False` and `True` for boolean values.
This code block will most likely disappear once the `in` check replaced the loop.

File: src/main.py [hunk 6]
```diff
@@ -63,9 +63,18 @@ def mensa(update, context):
     if not resp.ok:
         context.bot.send_message(chat_id=update.message.chat_id, text="I failed miserably. Disgrace!")
         return
+
     jsonData = json.loads(resp.content)
     for elem in jsonData:
-        context.bot.send_message(chat_id=update.message.chat_id, text=elem["name"])
+        list=elem["notes"]
+        isVeg = 0
+        for veg in list:
```
Comment (fried-ice) [inline src/main.py:71]: There is no need to loop over the elements of the `note` list. You can check via the mebership operator `in` - https://docs.python.org/3/reference/expressions.html#membership-test-operations.

Also note, that `veg` as an identifier is missleading.

**Review by fried-ice — CHANGES_REQUESTED** [review]

**Comment by MaxFleur** [comment]
Contains stuff that was merged already... clean up and will do another pull request in the future.
